packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, arrayOf } from 'prop-types';
+import { EasternArabic, WesternArabic } from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadLink } from './item';
+
+const MostReadItemColumns = {
+  columns: {
+    group0: 5,
+    group1: 5,
+    group2: 5,
+    group3: 5,
+    group4: 5,
+    group5: 5,
+  },
+};
+
+const MostReadRankColumns = {
+  item: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 1,
+    group3: 1,
+    group4: 1,
+    group5: 1,
+  },
+};
+
+const MostReadLinkColumns = {
```

Comment:
It'd be better to have these objects of props named to be `MostReadLinkProps` instead of `MostReadLinkColumns`, since they include props such as `item` - so not just columns.

Reply:
Resolved